### PR TITLE
close 40 date picker

### PR DESCRIPTION
**Pull request by ccfz**

### CODE REVIEW

File: app/controller/requests.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class Bnb < Sinatra::Base
   get '/requests' do
+    #puts "Lets see the bookings @ request cont"
```
Comment (robota-x) [inline app/controller/requests.rb]: remove comments

File: app/controller/requests.rb [hunk 2]
```diff
@@ -9,6 +11,10 @@ class Bnb < Sinatra::Base
       booking = Booking.get(params[:host_request_id])
       booking.status = 'confirmed'
       booking.save
+
+      space = Space.get(booking.space_id)
```
Comment (robota-x) [inline app/controller/requests.rb]: this var is not used or modified, so no need to call it and then save the db entry

File: spec/features/request_to_book_spec.rb [hunk 3]
```diff
@@ -1,6 +1,12 @@
 feature 'Request space' do
+
+  def get_date_range(start_date, end_date )
```
Comment (robota-x) [inline spec/features/request_to_book_spec.rb:3]: this is defined in the server.rb file (part of the program) so should not be needed

File: spec/unit/space_spec.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+require './app/model/space.rb'
```
Comment (robota-x) [inline spec/unit/space_spec.rb]: this should be already available with the following require chain: spec_helper < app.rb < data_mapper_setup.rb < model/space.rb

**Comment by robota-x** [comment]
done commenting, thank you for all the work :)

**Comment by lorenzoturrino** [comment]
:shipit:

**Comment by robota-x** [comment]
:shipit:

**Pull request merged**